spec/models/team_owner_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe TeamOwner, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I think you can remove this empty spec file according to the automatically generated notice (“add some examples to (**or delete**) …”).

Reply:
Same here